src/main/msp/msp.c

```diff
@@ -1545,7 +1545,7 @@ case MSP_NAME:
 #ifdef USE_GPS_RESCUE
     case MSP_GPS_RESCUE:
         sbufWriteU16(dst, gpsRescueConfig()->maxRescueAngle);
-        sbufWriteU16(dst, gpsRescueConfig()->initialAltitudeM);
+        sbufWriteU16(dst, gpsRescueConfig()->returnAltitudeM);  // actual this is 1 byte. But to maintain backwards compatibility with API 1.42 we use 2 bytes here.
```

Comment:
Because `returnAltitudeM` is `uint16_t`, I'd argue it is actually 2 bytes. IMO the comment adds confusion
```suggestion
        sbufWriteU16(dst, gpsRescueConfig()->returnAltitudeM);
```

Reply:
Thanks. Removed now as it was added because the confusion I got from reading other code which has now been fixed.